HOW_TO.md

```diff
@@ -41,15 +41,15 @@ only a few seconds to run.
 
 ```
 $ PYTHONPATH=. python3 -m pytest -v
-tests/unittest/benchmark_runner/regression/
+tests/unittest/benchmark_runner/common/workload_flavors/
 ============================== test session starts ===============================
 platform linux -- Python 3.9.5, pytest-6.2.2, py-1.10.0, pluggy-0.13.1 -- /usr/bin/python3
 cachedir: .pytest_cache
 rootdir: /home/rkrawitz/sandbox/benchmark-runner
 plugins: typeguard-2.10.0, venv-0.2
 collected 1 item
 
-tests/unittest/benchmark_runner/regression/test_golden_files.py::test_golden_files PASSED [100%]
+tests/unittest/benchmark_runner/common/workload_flavors/test_golden_files.py::test_golden_files PASSED [100%]
```

Comment:
Please add __init__.py in workload_flavors folder that it will be taken in the build package process.

Reply:
Done.